source/_includes/integrations/config_flow.md

```diff
@@ -0,0 +1,27 @@
+{% capture name %}{{ include.name | default: page.title }}{% endcapture %}
+
+## Configuration
```

Comment:
So, I'm a bit in doubt if I should add this heading in here or not...

Reply:
I think there should be a heading, but maybe "Setup" or something along those lines.

Reply:
The name of the heading was not the concern at this point. Its called `Configuration` everywhere right now 😉 

can always be changed.